config/application.rb

```diff
@@ -11,10 +11,6 @@
 # you've limited to :test, :development, or :production.
 Bundler.require(*Rails.groups)
 
-if !Rails.env.production? || ENV['HEROKU_APP_NAME'].present?
```

Comment:
This feels really dangerous

Reply:
Since we removed the `require: false` in https://github.com/alphagov/government-frontend/commit/4b0e39570e6401eefcc76c6b815017a396118256#diff-8b7db4d5cc4b8f6dc8feb7030baa2478L10 it's had this behaviour anyway.  I'll look into whether we can tune what's loaded in govuk_publishing_components.

Reply:
Since we removed the require: false in 4b0e395#diff-8b7db4d5cc4b8f6dc8feb7030baa2478L10 it's had this behaviour anyway. I'll look into whether we can tune what's loaded in govuk_publishing_components.